app/src/main/res/drawable/general_item_background_border_grey.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="rectangle">
+  <solid android:color="@color/component_color_profile_rename_activity_secondary_background_color" />
+  <stroke
+    android:width="1dp"
+    android:color="@color/color_def_black_12" />
+</shape>
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
Also, please use a component color for stroke too.

Reply:
done

Reply:
This seems to have been resolved.